firmware/quadruna/VC/src/jobs.h

```diff
@@ -16,3 +18,5 @@ void jobs_run1Hz_tick(void);
 void jobs_run100Hz_tick(void);
 void jobs_run1kHz_tick(void);
 void jobs_runCanRx_tick(void);
+
+void jobs_canRxCallback(const CanMsg* rx_msg);
```

Comment:
@liam-ilan Thought you might want this for tests? If not let me know and I'll probably just move it to be static in `hw_cans.c`.